src/CommunityToolkit.Maui/Views/DrawingView/Line.shared.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.ObjectModel;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The Line object is used to describe the lines that are drawn on a <see cref="IDrawingView"/>.
+/// </summary>
+public class Line : BindableObject, ILine
```

Comment:
Wodering if we don't want to inherit from `Element` instead... What do you think @VladislavAntonyuk ?

Reply:
if we assume it's a visual object, then yes, it make sense